### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_es_ES.properties [hunk 1]
```diff
@@ -77,13 +77,13 @@ ascanalpha.ldapinjection.knownimplementations=activedirectory\:openldap\:389dire
...
... cancelar\: demasiado tarde para cancelar\: no puede Canc EL\: Error de aserci\u00f3n\: Error de aserci\u00f3n (X)\: con proxy autorizaci\u00f3n es denegada\: Autorizaci\u00f3n proxy fallo (X)\: Contenido de sincronizaci\u00f3n de actualizaci\u00f3n requerido\: Contenido de sincronizaci\u00f3n de actualizaci\u00f3n requerida (X)\: No Operaci\u00f3n (X)\: Recursos LCUP de existencias\: Violaci\u00f3n LCUP de seguridad\: LCUP Los datos no v\u00e1lido\: no compatible LCUP Esquema\: LCUP Recargar requerido\: TXN especificar bien\: TXN ID no es v\u00e1lido\: no se puede contactar con el servidor LDAP\: error local\: error de codificaci\u00f3n\: La decodificaci\u00f3n de error\: Tiempo de espera agotado\: m\u00e9todo de autenticaci\u00f3n Desconocido\: filtro de b\u00fasqueda malo\: el usuario cancel\u00f3 la operaci\u00f3n\: Bad par\u00e1metro a una rutina de LDAP\: Fuera de la memoria\: Conectar error\: no se admite\: control no encontrado\: no hay resultados devueltos\: M\u00e1s resultados para volver\: Cliente Loop\: L\u00edmite de Referencia excedido\: API error desconocido\: error desconocido (extensi\u00f3n)\: Desconocido ( extensi\u00f3n privada) error\: error desconocido\n\u00a0
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_es_ES.properties]: I'm unconvinced that this was pulled from the library source as the comment outlines.

Comment (thc202) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_es_ES.properties]: Yeah, does no look like it. I'll open a PR to move these to other file for the time being.

Comment (thc202) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages_es_ES.properties]: Done in #1326.

**Comment by thc202** [comment]
Closing (we'll get a new (fresh) PR).

**Pull request closed without merging**